src/main/java/mServer/crawler/sender/br/BrWebAccessExecution.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * BrWebAccessExecution.java
+ * 
+ * Projekt    : MServer
+ * erstellt am: 07.10.2017
+ * Autor      : Sascha
+ * 
+ * (c) 2017 by Sascha Wiegandt
+ */
+package mServer.crawler.sender.br;
+
+import java.net.MalformedURLException;
+
+import com.google.gson.JsonSyntaxException;
+
+@FunctionalInterface
+public interface BrWebAccessExecution {
+    void run() throws JsonSyntaxException,
+                      IllegalArgumentException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)